assets/dev/scss/global/global.scss

```diff
@@ -1,7 +1,6 @@
 @import "../../../../assets/dev/scss/helpers/variables";
 @import "../../../../assets/dev/scss/editor/breakpoints";
 @import "dialog";
-@import "aspect_ratio";
```

Comment:
Will break new Pro versions that still uses the aspect-ratio classes.
Maybe keep those classes and just change the implementation, or at least keep this file for BC (2-3 major versions?)
And what about 3rd party? do they use those classes?

Reply:
@StyleShit In this case I'm pretty good with the backward compatibility. But I partially revert just to be safe until the next version. Please see the changes : https://github.com/elementor/elementor/pull/20718/commits/4e71425afb82b9bea3da2659fe0540e62481dfa3